curriculum/challenges/english/25-front-end-development/lab-lunch-picker-program/66db529d37ad966480ebb633.md

```diff
@@ -15,25 +15,36 @@ In this lab, you'll build a program that helps in managing lunch options. You'll
 
 1. You should create a `lunches` variable and assign it an empty array that will be used to store lunch items.
 
-2. You should create a function `addLunchToEnd` that takes a string parameter and adds it to the end of the `lunches` array and returns a string in the format: `"[Lunch Item] added to the end of the lunch menu."`
-
-3. You should create a function `addLunchToStart` that takes a string parameter and adds it to the start of the `lunches` array and returns a string in the format: `"[Lunch Item] added to the start of the lunch menu."`
-
-4. You should create a function `removeLastLunch` that removes the last lunch item from the `lunches` array and:
-   - If successful, returns a string in the format: `"[Lunch Item] removed from the end of the lunch menu."`
-   - If the `lunches` array is empty, returns a string: `"No lunches to remove."`
-
-5. You should create a function `removeFirstLunch` that removes the first lunch item from the `lunches` array and:
-   - If successful, returns a string in the format: `"[Lunch Item] removed from the start of the lunch menu."`
-   - If the `lunches` array is empty, returns a string: `"No lunches to remove."`
-
-6. You should create a function `getRandomLunch` that selects a random lunch item from the `lunches` array and:
-   - If successful, returns a string in the format: `"Randomly selected lunch: [Lunch Item]"`
-   - If the `lunches` array is empty, returns a string: `"No lunches available."`
-
-7. You should create a function `showLunchMenu` that:
-   - If there are items in the `lunches` array, returns a string in the format: `"Menu items: [Lunch Item], [Lunch Item]...`
-   - If the `lunches` array is empty, returns a string: `"The menu is empty."`
+2. You should create a function `addLunchToEnd` that takes an array as the first argument and a string as the second argument. The function should:
+    - Add the string to the end of the array.
+    - Log the string `"[Lunch Item] added to the end of the lunch menu."` to the console.
```

Comment:
```suggestion
    - Log the string `"[Lunch Item] added to the end of the lunch menu."` to the console where `[Lunch Item]` is the string passed to the function.
```

maybe it should specidied also in the other places, but here it's a must otherwise it's confusing